src/Okta.Sdk.UnitTests/FeaturesClientShould.cs

```diff
@@ -47,7 +46,7 @@ public async Task CallDependenciesEndpoint()
                 "[{\"Description\":null,\"Id\":null,\"name\":\"TestFeature\",\"Stage\":{\"State\":null,\"Value\":null},\"Status\":null,\"Type\":null}]";
             mockResponse.StatusCode = 200;
             _ = mockRequestExecutor
-                .GetAsync(dependenciesEndpoint, Arg.Any<Dictionary<string, string>>(), Arg.Any<CancellationToken>())
+                .ExecuteRequestAsync(Arg.Any<HttpRequest>(), Arg.Any<CancellationToken>())
```

Comment:
I could see you still use `PostAsync` above, but you are replacing `GetAsync` by `ExecuteRequestAsync` here, Is `GetAsync` no longer available?

Reply:
ExecuteRequestAsync will delegate to the appropriate method (PostAsync, GetAsync, etc.) based on the verb specified in the HttpRequest.  There are different layers delegating downward, this change was necessary for the logic of this test to represent the new underlying behavior.